src/graphql/query/checkout/checkoutStatus.graphql

```diff
@@ -0,0 +1,11 @@
+query checkoutStatus($transactionId: String!, $visitorId: String!) {
+	checkoutStatus(transactionId: $transactionId, visitorId: $visitorId) {
```

Comment:
Is this being used?

Reply:
Follow commits will use this in the polling function. Unless I end up setting that up in kv-shop, in which case I can remove this file.